ldapadmin/src/main/java/org/georchestra/ldapadmin/ws/backoffice/users/UsersController.java

```diff
@@ -691,59 +693,50 @@ private Account modifyAccount(Account account, ServletInputStream inputStream) t
 	 */
 	private Account createAccountFromRequestBody(ServletInputStream is) throws IllegalArgumentException, IOException {
```

Comment:
Why create account in user controller ?
Nothing redondant with `create` method of `NewAccountFormController` ?